packages/superfluidDaoFront/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import "./App.css";
+import NavBar from "./components/NavBar";
+import LandingComponent from "./pages/Landing";
+import CreateDao from "./pages/CreateDao";
+import YoursDao from "./pages/YoursDao";
+import Dashboard from "./pages/Dashboard";
+import AboutUs from "./pages/AboutUs";
+import { Routes, Route } from "react-router-dom";
+
+const App = () => {
+  return (
+    <div className="app-container">
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<LandingComponent />} />
+        <Route path="/create-dao" element={<CreateDao />} />
+        <Route path="/yours-dao" element={<YoursDao />} />
```

Comment:
```suggestion
        <Route path="/your-daos" element={<YourDaos />} />
```
yours dao c'est pas anglais